module/documents/actor.mjs

```diff
@@ -90,12 +90,15 @@ export default class ActorEd extends Actor {
    */
   async rollAttribute( attributeId, options = {} ) {
     const attributeStep = this.system.attributes[attributeId].step;
+    const attribute = `${game.i18n.localize( ED4E.attributes[attributeId].label )} Test`
+    const sourceActor = this.name;
+    let targetActors = this.targetNames()
     const edRollOptions = new EdRollOptions( {
       testType: "action",
       step: { base: attributeStep },
       karma: { pointsUsed: this.system.karma.useAlways ? 1 : 0, available: this.system.karma.value, step: this.system.karma.step },
       devotion: { available: this.system.devotion.value, step: this.system.devotion.step },
-      chatFlavor: `${game.i18n.localize( ED4E.attributes[attributeId].label )} Test`,
+      chatFlavor: sourceActor + " " + game.i18n.localize( "ED.Rolls.rolls" ) + " " + attribute + targetActors,
```

Comment:
Try to avoid string concat, especially in localization. This is another use case for the `i18n.format(...)` method

Reply:
solved by factoring out the targets. and yes